content/aips/AAVE-V3-POLYGON-CAPS-UPDATES-20230518.md

```diff
@@ -0,0 +1,64 @@
+---
+title: [ARFC] wMATIC Supply & Borrow Cap Increase Polygon v3
+discussions: https://governance.aave.com/t/arfc-wmatic-supply-borrow-cap-increase-polygon-v3/13095
+shortDescription: Increase Supply & Borrow Cap wMATIC on Polygon v3
+author: Llama (Fermin Carranza, TokenLogic)
+created: 2023-05-18
+---
+
+# Summary
+
+This AIP proposes increasing the wMATIC Supply and Borrow Cap on Polygon v3.
+
+# Abstract
+
+With strong deposit flows from the Balancer bb-a-wMATIC integration and the continual growth of the yield maximising strategy, the Supply and Borrow Cap of wMATIC are both experiencing strong utilisation.
+
+This AIP enables the continued growth of wMATIC deposits as Balancer Liquidity Providers migrate to the new gauges and users continue to migrate from Aave v2 to v3. As wMATIC deposits grow so does the demand for borrowing wMATIC. Subsidised borrowing costs has lead to significant demand for wMATIC and TVL growth for Aave.
+
+Increasing the Supply and Borrow Caps as outlined will continue to users of yield maximising strategies.
+
+# Motivation
+
+With several Balancer liquidity pools depositing wMATIC into Aave v3 on Polygon, the utilisation of the Supply Cap has increased to 100%. This is currently preventing the Balancer linear pools from depositing into the wMATIC reserve. 
+
+Currently, there is around [5,294,939](https://app.balancer.fi/#/polygon/pool/0x8159462d255c1d24915cb51ec361f700174cd99400000000000000000000075d) and [1,228,487](https://app.balancer.fi/#/polygon/pool/0xb20fc01d21a50d2c734c4a1262b4404d41fa7bf000000000000000000000075c) units of wMATIC not yet migrated to the new gauges. The wMATIC Supply Cap is at 100% utilisation.
+
+There is also [15.13M](https://app.aave.com/markets/?marketName=proto_polygon) units of wMATIC on Aave v2 that has not yet migrated to v3. 
+
+Collectively, this represents around 21.65M units of wMATIC relative to a Supply Cap. This AIP intends to provide sufficent capacity for the 21.65M units to be deposited into the wMATIC reserve by increasing the Supply Cap from 66M units to 90M units, a 24M unit increase. The utilisation of the pool will be 73.33% after the Supply Cap is implemented.
+
+![](../assets/AAVE-V3-POLYGON-CAPS-UPDATES-20230518/utilization.png)
+
+With many teams are offering yield maximising strategies, there is continued strong demand for borrowing wMATIC. The current Borrow Cap has reached 74.50% utilisation. The newly proposed Borrow Cap 50M units is equivalent to 25% increase and represents 62.5% of the Supply Cap.
+
+![](../assets/AAVE-V3-POLYGON-CAPS-UPDATES-20230518/borrowinfo.png)
```

Comment:
this won't work. I'd recommend to remove the image as it does not provide any additional value.
```suggestion
```

> When linking to an image in the AIP, you must use the full url for the raw content, for example https://raw.githubusercontent.com/aave/aip/main/content/assets/[UPPERCASE_IDENTIFIER]/image.png.